crates/net/network/src/metrics.rs

```diff
@@ -244,3 +246,45 @@ pub struct EthRequestHandlerMetrics {
     /// Number of received bodies requests
     pub(crate) received_bodies_requests: Counter,
 }
+
+/// Eth67 announcement metrics, track entries by TxType
+#[derive(Metrics)]
+#[metrics(scope = "network.transaction_fetcher")]
+pub struct AnnouncedTxSizeMetrics {
```

Comment:
the code is measuring tx types not sizes, so a more appropriate name
```suggestion
pub struct AnnouncedTxTypesMetrics {
```